pyproject.toml

```diff
@@ -0,0 +1,199 @@
+[project]
+# This is the name of your project. The first time you publish this
+# package, this name will be registered for you. It will determine how
+# users can install this project, e.g.:
+#
+# $ pip install sampleproject
```

Comment:
can't you change `sampleproject` to `quantus`? While reading the comments, this file feels a bit unfinished.

Reply:
The file and all the comments were taken from https://github.com/pypa/sampleproject/blob/main/pyproject.toml.

Reply:
done

Reply:
> can't you change `sampleproject` to `quantus`? While reading the comments, this file feels a bit unfinished.

done